swingmusic/umbrel-app.yml

```
@@ -0,0 +1,26 @@
+manifestVersion: 1
+id: swingmusic
+name: Swing Music
+tagline: Self-hosted music streaming (Spotify alternative)
+icon: ""
+category: media
+version: "1.0.0"
```

Change to installed version.
```suggestion
version: "2.0.8"
```